### PR TITLE
CN-33 - need to help

### PR DESCRIPTION
**Pull request by Nemo-Git-Hub**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -274,6 +274,321 @@ <h3 class="text-2 text-center color-accent">
       </div>
     </section>
 
+    <section class="out-production">
+      <div class="container">
+        <div class="row">
+          <div class="col">
+            <p class="text-2 text-center text-uppercase">наша продукція</p>
+          </div>
+        </div>
```
Comment (asd-ez) [inline index.html]: Here don't need `row` and `col`.

File: index.html [hunk 2]
```diff
@@ -274,6 +274,321 @@ <h3 class="text-2 text-center color-accent">
...
+
+        <div class="row">
+          <div class="col-12 col-lg-4 mx-auto mb-5">
+            <div class="out-production-block w-100 d-flex flex-column">
+              <img
+                src="src/assets/img/gorbusha-225.png"
+                alt="caviar"
+                class=""
```
Comment (asd-ez) [inline index.html]: Remove the empty `class` attribute.

File: index.html [hunk 3]
```diff
@@ -274,6 +274,321 @@ <h3 class="text-2 text-center color-accent">
...
+              <p class="gilroy">
+                <span class="fw-bold">Склад:</span> ікра горбуші, сіль, олія<br /><br />
+                <span class="fw-bold">Тара:</span> скло<br /><br />
+                <span class="fw-bold">Колір:</span>
+                яскраво-жовтогарячий<br /><br />
+                <span class="fw-bold">Розмір ікринок:</span> 4 мм у діаметрі<br /><br />
+                <span class="fw-bold">Консистенція:</span> суха. Не більше ніж
+                5-7% рідини<br /><br />
+                <span class="fw-bold">Термін придатності до споживання:</span>
+                4 міс. у закритому вигляді, 2 тижні у відкритому
```
Comment (asd-ez) [inline index.html]: Don't use the `<br>` tag, please.

File: index.html [hunk 4]
```diff
@@ -274,6 +274,321 @@ <h3 class="text-2 text-center color-accent">
...
+                class="out-production-block-weight d-flex w-100 align-items-center"
+              >
+                <div class="w-100">
+                  <p class="gilroy text-7 mb-0">
+                    <span class="text-6 fw-semibold mr-4">Вага</span> (нетто)
+                  </p>
+                </div>
+                <div class="dropdown">
+                  <a
+                    class="btn btn-btn-outline-dark dropdown-toggle out-production-block-gramm gilroy fw-semibold text-4"
```
Comment (asd-ez) [inline index.html]: If you have a custom class(here `out-production-block-gramm`), write his last or first for all tags.

File: index.html [hunk 5]
```diff
@@ -274,6 +274,321 @@ <h3 class="text-2 text-center color-accent">
...
+                <span class="fw-bold">Склад:</span> ікра лосося, сіль, олія<br /><br />
+                <span class="fw-bold">Тара:</span> скло<br /><br />
+                <span class="fw-bold">Колір:</span>
+                яскраво-жовтогарячий<br /><br />
+                <span class="fw-bold">Розмір ікринок:</span> 4 мм у діаметрі<br /><br />
+                <span class="fw-bold">Консистенція:</span> суха. Не більше ніж
+                5-7% рідини<br /><br />
+                <span class="fw-bold">Термін придатності до споживання:</span>
+                4 міс. у закритому вигляді, 2 тижні у відкритому
+              </p>
```
Comment (asd-ez) [inline index.html]: Don't use the `<br>` tag, please.

File: index.html [hunk 6]
```diff
@@ -274,6 +274,321 @@ <h3 class="text-2 text-center color-accent">
...
+              <p class="gilroy text-center mt-4">Залишилось банок: 5 шт.</p>
+              <button type="button" class="btn btn-primary text-5">
+                Замовити зараз
+              </button>
+            </div>
+          </div>
+
+          <div class="col-12 col-lg-4 mx-auto mb-5">
+            <div class="out-production-block w-100 d-flex flex-column">
+              <img src="src/assets/img/losos-225.png" alt="caviar" class="" />
```
Comment (asd-ez) [inline index.html]: Remove the empty class attribute.

File: src/scss/_variables.scss [hunk 7]
```diff
@@ -2,16 +2,15 @@
 $font-color-black: #151515;
 $background-color-white: #f5f5f5;
 $background-color-white-clean: #fff;
-// $grey: #969696;
+$background-color-weight: #f4f4ff;
```
Comment (asd-ez) [inline src/scss/_variables.scss:5]: Color for what? Rewrite the variable for more understanding, like previously.

**Review by asd-ez — APPROVED** [review]

File: src/scss/_variables.scss (hunk 7 above)
Comment (Nemo-Git-Hub) [reply src/scss/_variables.scss:5]: I need an advice

**Pull request merged**